Remotion/Web/Development.WebTesting.IntegrationTests/MouseTest.cs

```diff
@@ -119,6 +119,7 @@ public void Test_Hover ()
       var home = Start();
 
       var hoverElement = home.Scope.FindId (c_hoverDivID);
+      Thread.Sleep (250);

```

Comment:
Okay?! If it works, it works. I know how Selenium testing can be 👯

Reply:
so sad. of course, we also do this in line 124... Never use mouse, I guess.